### PR TITLE
Release SonarDelphi plugin v1.12.0

### PR DESCRIPTION
**Pull request by Cirras**
SonarDelphi v1.12.0 has been released:

- [Release Notes](https://github.com/integrated-application-development/sonar-delphi/releases/tag/v1.12.0)
- [SonarCloud](https://sonarcloud.io/project/overview?id=integrated-application-development_sonar-delphi)

Please add it to the marketplace.

Thanks!

### CODE REVIEW

**Comment by ganncamp** [comment]
The new releases are in progress & with them, the format of these version blocks will change slightly. So I'm going to hold off processing this until that's complete. It should be later this week.

File: communitydelphi.properties [hunk 1]
```diff
@@ -1,14 +1,20 @@
...
-archivedVersions=1.0.0,1.1.0,1.2.0,1.3.0,1.4.0,1.5.0,1.6.0,1.7.0,1.8.0,1.9.0,1.10.0
-publicVersions=1.11.0
+archivedVersions=1.0.0,1.1.0,1.2.0,1.3.0,1.4.0,1.5.0,1.6.0,1.7.0,1.8.0,1.9.0,1.10.0,1.11.0
+publicVersions=1.12.0
 
 defaults.mavenGroupId=au.com.integradev.delphi
 defaults.mavenArtifactId=sonar-delphi-plugin
 
+1.12.0.description=Improvements to intrinsics, parsing improvements, bug fixes
+1.12.0.sqVersions=[9.9,LATEST]
```
Comment (ganncamp) [inline communitydelphi.properties:11]: With the release of SonarQube Community Build and SonarQube Server, this changes to:
```
1.12.0.sqs=[10.8,LATEST]
1.12.0.sqcb=[24.12,LATEST]
1.12.0.sqVersions=[9.9,10.7]
```

**Review by ganncamp — CHANGES_REQUESTED** [review]

File: communitydelphi.properties (hunk 1 above)
Comment (Cirras) [reply communitydelphi.properties:11]: Thanks, I've updated to the new format.

**Comment by Cirras** [comment]
Updated to the new versioning format.

**Review by ganncamp — APPROVED** [review]

**Pull request merged**